README.md

```diff
@@ -129,6 +127,16 @@ Find your maven `settings.xml` file. On Linux it should be at `~/.m2/settings.xm
 </repository>
 ```
 
+### Step 3.9: (If using Invoke) create a .env file
```

Comment:
Not sure where the .9 comes from, but rather than having "If using invoke" sections, it might be clearer to just have a line above saying something like "Please see the section on "invoke" below for tools that can streamline this process".